docs/source/topics/proc_changing-the-selected-preset.adoc

```diff
@@ -0,0 +1,31 @@
+[id="changing-the-selected-preset_{context}"]
+= Changing the selected preset
+
+[role="_abstract"]
+You can change the container runtime used for the {prod} instance by selecting the desired preset.
+
+On {msw} and {mac}, you can change the selected preset using the system tray or command line interface.
+On Linux, use the command line interface.
+
+[IMPORTANT]
+====
+You cannot change the preset of an existing {prod} instance.
+Preset changes are only applied when a {prod} instance is created.
+To enable preset changes, you must delete the existing instance and start a new one.
```

Comment:
Not sure how this will stand out in the documentation. It briefly touches on 'delete'. Will have to check the rendered version.

Reply:
I placed the content under an "important" admonition to (hopefully) encourage careful reading of it. Do you think we should make this more visible somehow?